src/pal/src/CMakeLists.txt

```diff
@@ -68,7 +68,13 @@ endif()
 # turn off capability to remove unused functions (which was enabled in debug build with sanitizers)
 set(CMAKE_SHARED_LINKER_FLAGS_DEBUG "${CMAKE_SHARED_LINKER_FLAGS_DEBUG} -Wl,--no-gc-sections")
 
-add_compile_options(-fno-builtin)
+# NetBSD requires a compiler (built-in) version of alloca(3)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL NetBSD)
+  add_compile_options(-fno-builtin)
```

Comment:
@jkotas I have looked through the history of the file before we moved to github and I really don't know the reason. The option came in with the initial checkin of another CMakeLists.txt file from @sergiy-k (it was later migrated to the current one).
@sergiy-k, do you have any idea why it was added?